corehq/apps/users/templates/users/web_users.html

```diff
@@ -1,365 +1,697 @@
-{% extends "users/base_template.html" %}
+{% extends 'style/bootstrap3/base_section.html' %}
+{% load crispy_forms_tags %}
 {% load i18n %}
 {% load hq_shared_tags %}
+{% load djangular_tags %}
 
 {% block js %}{{ block.super }}
     <script src="{% static 'hqwebapp/js/main.js' %}"></script>
-    <script src="{% static 'style/lib/knockout_plugins/knockout_mapping.ko.min.js' %}"></script>
-    <script src="{% static 'style/ko/knockout_bindings.ko.js' %}"></script>
     <script src="{% static 'users/js/roles.js' %}"></script>
+    <script src="{% static 'users/js/web_users_list.ng.js' %}"></script>
```

Comment:
@orangejenny @biyeun should this be `new_static` instead of `static`? we're trying to figure out the root cause of this bug http://manage.dimagi.com/default.asp?181567